ext/lmdb-safe/lmdb-safe.hh

```diff
@@ -268,62 +277,62 @@ inline string_view MDBOutVal::getNoStripHeader<string_view>() const
 class MDBInVal
 {
 public:
-  MDBInVal(const MDBOutVal& rhs): d_mdbval(rhs.d_mdbval)
+  MDBInVal(const MDBOutVal& rhs) :
+    d_mdbval(rhs.d_mdbval)
   {
   }
 
 #ifndef DNSDIST
-  template <class T,
-            typename std::enable_if<std::is_integral<T>::value,
-                                    T>::type* = nullptr>
-  MDBInVal(T i)
+  template <class T>
+  MDBInVal(T rhs)
```

Comment:
is the `is_integral` check gone on purpose?

Reply:
Yes. The reason is that the upcoming `uint128_t` type is not going to match `is_integral`.